WmgmtVehicle/examples/example.jsonld

```diff
@@ -0,0 +1,25 @@
+{
+  	"id":"https://smart-data-models.github.io/dataModel.WasteManagement/WmgmtVehicle/schema.json",
```

Comment:
for this example, this uri is not the usual one (it is pointing the schema origin), I'll change its value.